xDrip/Extensions/TimeFrameSelectionView.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  TimeFrameSelectionView.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 20.03.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import UIKit
+
+final class TimeFrameSelectionView: UIView {
+    
+    typealias SegmentChangeCallback = (Int) -> ()
+    
+    private let backgroundViewCornerRadius: CGFloat = 9.0
+    private let selectedSegmentViewCornerRadius: CGFloat = 7.0
+    
+    private let separatorWidthConstraintConstant: CGFloat = 1.0
+    private let separatorHeightConstraintConstant: CGFloat = 12.0
+    
+    private let labelRegularFont = UIFont.systemFont(ofSize: 16.0, weight: .regular)
```

Comment:
Rename it to something like `labelSelectedFont` and `labelDeselectedFont`. Because `Regular` and `Medium` could change over time to something else, but `selected`/`deselected` clearly specify the goal of these fonts.